src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,43 @@
+package codeu.controller;
+
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import java.time.Instant;
+import codeu.model.data.Conversation;
+import codeu.model.data.User;
+import code.model.data.Message;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+
+
+public class AdminServlet extends HttpServlet {
+    List<String> admin = Arrays.asList("anAdmin", "Admin1", "Admin2");
+    private UserStore userStore;
+    private ConversationStore conversationStore;
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException, ServletException {
+        request.getRequestDispatcher("/WEB-INF/view/admin.jsp").forward(request, response);
```

Comment:
We may want to add an if statement that checks if the user is an admin (before line 26). As it stands non-admin users will be able to view the admin page. If the user is not an admin then we can just redirect them to a different page or render a warning in the jsp. Feel free to implement it however you like.

Reply:
thank you for the comment, and now my question is if i should add a feature in the menu so that the admin can access the page without typing in themselves?